moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -315,12 +324,14 @@ class Robot
 
   void updateState(const sensor_msgs::JointStateConstPtr& msg);
 
+  void updateCurrentState(const sensor_msgs::JointStateConstPtr& msg);
 
   // Robot internal objects
   std::map<std::string, TrajectoryInfo> traj_info_map_;
 
   // MoveIt objects
   mutable moveit::core::RobotStatePtr robot_state_;
```

Comment:
I realize this is the name that I chose, but now that there is two "robot states", the names should indicate the difference.  That isn't the case right now.  Can you think of new names?